onchain/rollups/contracts/consensus/quorum/Quorum.sol

```diff
@@ -39,17 +48,15 @@ contract Quorum is AbstractConsensus, AccessControlEnumerable, PaymentSplitter {
         uint256[] memory _shares,
         IHistory _history
     ) PaymentSplitter(_validators, _shares) {
-        // Iterate through the array of validators,
-        // and grant to each the validator role.
+        // Add the array of validators into the quorum
         for (uint256 i; i < _validators.length; ++i) {
-            grantRole(VALIDATOR_ROLE, _validators[i]);
+            validators[_validators[i]] = true;
         }
-
-        // Set history.
+        quorumSize = _validators.length;
         history = _history;
     }
 
-    /// @notice Submits a claim for voting.
+    /// @notice Submits (Votes) a claim.
```

Comment:
Maybe we could drop the "Submit", and instead keep "Vote _for_ a claim to be submitted".